run_differential_expression_experiments.sh

```diff
@@ -20,5 +20,4 @@ fi
 
 # Run differential expression scripts
 Rscript 1A-detect_differentially_expressed_genes.R --cancer_type $cancer_type --subtype_vs_others $subtype_vs_others --subtype_vs_subtype $subtype_vs_subtype --ncores $ncores
-Rscript 2A-plot_DE_results.R --cancer_type $cancer_type --subtype_vs_others $subtype_vs_others --subtype_vs_subtype $subtype_vs_subtype
```

Comment:
Suggests to me that `3A` needs to get renumbered

Reply:
Renamed to make everything consistent. Now have 1A and 2A analysis scripts, matched by 1A and 2A plotting scripts.